src/index.js

```diff
@@ -2,26 +2,16 @@
  * @flow
  */
 
-// example of generating HTML with js
-function renderSidebar() {
-  const sidebarContents = `
-    <h2 class="email-header">Inbox</h2>
-    <ul class="email-list">
-      <li>
-        <button class="email-item" type="button">
-          <div class="sender-details">
-            <p>Email sender</p>
-            <span>Timestamp</span>
-          </div>
-          <p class="email-subject">Email1 subject</p>
-          <p>Email snippet</p>
-        </button>
-      </li>
-    </ul>
-  `;
+import store from './store/index';
```

Comment:
This is probably just personal preference, but `import store from './store'` (without the trailing `/index`) works here, right?

Reply:
Blah... shows what I know about ES6